styles/style-main.css

```diff
@@ -461,3 +495,83 @@ h1 {
   font-weight: 600;
   padding-inline-start: 35px;
 }
+
+.contact-me {
+  display: flex;
+  margin-top: 50px;
+  background-image: url(/images/mobile_geometry_1.png);
+  background-repeat: no-repeat;
```

Comment:
Nice job implementing most changes suggested. You have one thing left out;

- [ ] The file path to the background image for this section is broken. I think you can fix it by changing it to 

 ` background-image: url(../images/mobile_geometry_1.png);`